.github/workflows/linux-build.yml

```
@@ -11,13 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 name: Linux Build using GCC
-
 on:
   push:
-    branches:
-      - "main"
+    branches: [presto-0.293-clp-connector]
```

```suggestion
    branches:
      - "presto-0.293-clp-connector"
```

To reduce the size of the diff.